airflow/providers/google/suite/hooks/drive.py

```diff
@@ -128,17 +128,19 @@ def get_media_request(self, file_id: str) -> HttpRequest:
         request = service.files().get_media(fileId=file_id)
         return request
 
-    def exists(self, folder_id: str, file_name: str, drive_id: Optional[str] = None):
+    def exists(self, folder_id: str, file_name: str, drive_id: Optional[str] = None, include_trashed: Optional[bool] = False):
```

Comment:
Why Optional? I think this can simply take `bool`.

Reply:
I think not including the trashed files is a sensible default.  Keeping it optional will also avoid breaking existing code that uses this function on the hook.

Reply:
I think having it not consider the trashed files is a sensible default.  Also, this will avoid breaking existing code that calls this method on the hook.

Reply:
That explains `= False`, but not `Optional`. This can be achieved with `include_trashed: bool = False`.